zone/attack.cpp

```diff
@@ -1843,6 +1843,15 @@ bool Client::Death(Mob* killerMob, int64 damage, uint16 spell, EQ::skills::Skill
 		exploss *= zone->level_exp_mod[GetLevel()].ExpMod;
 	}
 
+	if (exploss > 0 && RuleB(Character, DeathKeepLevel)) {
+		int32 totalExp = GetEXP();
```

Comment:
`snake_case` eg `total_exp` for locally scoped variables

Same for below variables